crawl-ref/source/webserver/ws_handler.py

```diff
@@ -658,8 +687,13 @@ def go_admin(self):
 
     def get_rc(self, game_id):
         if game_id not in config.games: return
-        with open(self.rcfile_path(game_id), 'r') as f:
-            contents = f.read()
+        path = self.rcfile_path(game_id)
+        try:
+            with open(path, 'r') as f:
+                contents = f.read()
+        # IOError for py2, OSError for py3
+        except (OSError, IOError):
+            contents = ''
```

Comment:
should maybe log an error if this happens? Or do some servers not initialize rc files?

Reply:
ok, this comment was largely a byproduct of not reading the PR by commit.

Reply:
CPO doesn't. All dgamelaunch-config servers do, but with this change they won't have to create 10+ 0-byte files per user sign-up 😊